kustomizations/apps/data-hub-configs/web-api-data-pipeline.config.yaml

```diff
@@ -370,3 +370,19 @@ webApi:
       recordTimestamp:
         itemTimestampKeyFromItemRoot:
           - date
+  # EuropePMC: Articles that are evaluated by Sciety (via LabsLink)
+  - dataset: '{ENV}'
+    table: 'europepmc_sciety_evaluated'
```

Comment:
Or maybe `europepmc_sciety_evaluated_via_labslink` to make it more explict?

Reply:
It would be good, if there is another way to evaluate.

Reply:
LabsLink is how we get the evaluations into EuropePMC. But they also started integrating DocMaps.